src/provider/provider_level_zero.c

```diff
@@ -211,6 +211,49 @@ static umf_result_t ze2umf_result(ze_result_t result) {
     }
 }
 
+static umf_result_t ze_init_drivers(void *lib_handle, const char *lib_name) {
+    ze_result_t (*zeInitDriversFunc)(uint32_t *, ze_driver_handle_t *,
+                                     ze_init_driver_type_desc_t *);
+    *(void **)&zeInitDriversFunc =
+        utils_get_symbol_addr(lib_handle, "zeInitDrivers", lib_name);
+    if (!zeInitDriversFunc) {
+        return UMF_RESULT_ERROR_DEPENDENCY_UNAVAILABLE;
+    }
+
+    ze_init_driver_type_desc_t desc = {
+        .stype = ZE_STRUCTURE_TYPE_INIT_DRIVER_TYPE_DESC,
+        .pNext = NULL,
+        .flags = UINT32_MAX};
```

Comment:
If you want only GPU Drivers, you need to set this to _GPU otherwise you will get back GPUs and NPUs.

Reply:
Done.